META-INF/maven/org.openmrs.module/isanteplus-api/pom.properties

```diff
@@ -0,0 +1,5 @@
+#Generated by Maven
+#Thu Sep 29 10:21:37 CST 2016
+version=1.0-SNAPSHOT
+groupId=org.openmrs.module
+artifactId=isanteplus-api
```

Comment:
this file is in the wrong place, what happened to your repo to have these files misplaced